test/shared/BaseVault.behaviour.ts

```diff
@@ -588,6 +588,7 @@ export function shouldBehaveLikeBaseVault(ctx: () => BaseVaultBehaviourContext):
 }
 
 export const testAmounts = (amount: number, assetDecimals = 18, vaultDecimals = 18): Amounts => {
+    vaultDecimals = assetDecimals
```

Comment:
I think we can remove the `vaultDecimals` param since we don't use it and this line makes it redundant